packages/xs-extra-dummy/xs-toolstack.master/opam

```diff
@@ -54,7 +54,7 @@ depends: [
   "xen-api-client"
   "xen-api-client-async"
   "xen-api-sdk"
-  "xenctrl"
+  "xenctrl" {= "0.1.0"}
```

Comment:
Is this breaking the tests?

```
#=== ERROR while compiling xenctrl.dummy ======================================#

# context              2.0.7 | linux/x86_64 | ocaml-base-compiler.4.08.1 | file:///mnt

# path                 ~/.opam/4.08/.opam-switch/build/xenctrl.dummy

# command              /usr/bin/rsync -avz /usr/lib64/ocaml/xenctrl/ /home/opam/.opam/4.08/lib/xenctrl/

# exit-code            23

# env-file             ~/.opam/log/xenctrl-961-0c8b92.env

# output-file          ~/.opam/log/xenctrl-961-0c8b92.out

### output ###

# sending incremental file list

# rsync: change_dir "/usr/lib64/ocaml/xenctrl" failed: No such file or directory (2)

# created directory /home/opam/.opam/4.08/lib/xenctrl

# 

# sent 20 bytes  received 68 bytes  176.00 bytes/sec

# total size is 0  speedup is 0.00

# rsync error: some files/attrs were not transferred (see previous errors) (code 23) at main.c(1207) [sender=3.1.3]
```

Reply:
I suspect so - why do we need the version constraint?

Reply:
this is confusing; i had this failure on my machine, and adding that constraint fixed it- i'll investigate

Reply:
i've versioned `xenctrl.dummy` as version 0 and the real xenctrl as v 1, so hopefully opam will pick the correct one now